content/docs/run-api-server/ingestion.mdx

```diff
@@ -162,15 +162,15 @@ export RETRY_BACKOFF_SECONDS=20
 If Horizon was previously running, first ensure it is stopped. Then, run the following commands in parallel:
 
 1. `stellar-horizon db reingest range --parallel-workers=64 1 16999999`
-2. `stellar-horizon db reingest range --parallel-workers=24 17000000 <latest_ledger>`
+2. `stellar-horizon db reingest range --parallel-workers=20 17000000 <latest_ledger>`
 
 (Where you can find `<latest_ledger>` under [SDF Horizon's](https://horizon.stellar.org/) `core_latest_ledger` field.)
 
-When saturating an RDS instance with 15K IOPS capacity:
+When saturating a database instance with 15K IOPS capacity:
```

Comment:
I don't think it's bad to be specific about RDS since (AFAIU) the IOPS metrics are amazon specific?

Reply:
I changed another place where we mention IOPS to be cloud provider agnostic too, other than that the metric should be available in most cloud providers (or even on bare metal)

Reply:
Ah, I didn't know the term is applicable to every cloud provider in the same way. OK